rag-sql-router/app.py

```
@@ -0,0 +1,454 @@
+            )
+
+
+if __name__ == "__main__":
+    # Clean up any temporary directories on exit
+    if hasattr(st.session_state, "temp_dir") and os.path.exists(
+        st.session_state.temp_dir
+    ):
+        try:
+            os.rmdir(st.session_state.temp_dir)
+        except:
+            pass
```

Ruff (0.12.2)

Use `contextlib.suppress(BaseException)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(BaseException)`

(SIM105)